assets/xml/objects/object_kamejima.xml

```diff
@@ -5,68 +5,73 @@
         <Texture Name="object_kamejima_Tex_000AD8" OutName="tex_000AD8" Format="i8" Width="64" Height="64" Offset="0xAD8" />
         <TextureAnimation Name="object_kamejima_Matanimheader_001AF0" Offset="0x1AF0" />
         <Collision Name="object_kamejima_Colheader_002328" Offset="0x2328" />
-        <!-- <Blob Name="object_kamejima_Blob_002354" Size="0x1C" Offset="0x2354" /> -->
+        <TextureAnimation Name="gTurtleEmptyAnimatedMat" Offset="0x2360"/>
```

Comment:
The suffix for this is `TexAnim`: https://github.com/zeldaret/ZAPD/blob/master/docs/zapd_extraction_xml_reference.md#textureanimation